src/tax-lot/tax-lot.controller.ts

```diff
@@ -36,44 +36,42 @@ export class TaxLotController {
   constructor(private readonly taxLotService: TaxLotService) {}
 
   @Get("/:bbl")
-  @UsePipes(new ZodValidationPipe(getTaxLotByBblPathParamsSchema))
-  async findDetailsByBbl(@Param() params: GetTaxLotByBblPathParams) {
-    return this.taxLotService.findTaxLotByBbl(params.bbl);
+  @UsePipes(new ZodValidationPipe(findTaxLotByBblPathParamsSchema))
+  async findDetailsByBbl(@Param() params: FindTaxLotByBblPathParams) {
+    return this.taxLotService.findByBbl(params.bbl);
   }
 
   @Get("/:bbl/geojson")
-  @UsePipes(new ZodValidationPipe(getTaxLotGeoJsonByBblPathParamsSchema))
+  @UsePipes(new ZodValidationPipe(findTaxLotGeoJsonByBblPathParamsSchema))
   async findTaxLotByBblGeoJson(
-    @Param() params: GetTaxLotGeoJsonByBblPathParams,
+    @Param() params: FindTaxLotGeoJsonByBblPathParams,
   ) {
-    return this.taxLotService.findTaxLotByBblGeoJson(params.bbl);
+    return this.taxLotService.findByBblGeoJson(params.bbl);
   }
 
   @Get("/:bbl/zoning-districts")
   @UsePipes(
-    new ZodValidationPipe(getZoningDistrictsByTaxLotBblPathParamsSchema),
+    new ZodValidationPipe(findZoningDistrictsByTaxLotBblPathParamsSchema),
   )
   async findZoningDistrictByTaxLotBbl(
-    @Param() params: GetZoningDistrictsByTaxLotBblPathParams,
+    @Param() params: FindZoningDistrictsByTaxLotBblPathParams,
   ) {
-    return this.taxLotService.findZoningDistrictByTaxLotBbl(params.bbl);
+    return this.taxLotService.findZoningDistrictByBbl(params.bbl);
   }
 
   @Get("/:bbl/zoning-districts/classes")
   @UsePipes(
-    new ZodValidationPipe(getZoningDistrictClassesByTaxLotBblPathParamsSchema),
+    new ZodValidationPipe(findZoningDistrictClassesByTaxLotBblPathParamsSchema),
   )
   async findZoningDistrictClassByTaxLotBbl(
```

Comment:
```suggestion
  async findZoningDistrictClassesByBbl(
```

The corresponding service and repositories should also make "classes" plural